.github/workflows/docs.yml

```diff
@@ -1,5 +1,6 @@
 name: Docs Publish
 on:
+  workflow_dispatch:
```

Comment:
I think this would only be used in instances where the docs were built and deployed for a release but somehow did not deploy successfully or need to be re-deployed? I'm fine with the added convenience and I don't think misclicks will be an issue.

Reply:
In that case, you could use GitHub's UI to rerun the failed action.

I think this somewhat depends on how much you want a policy of Everyone Is An Adult Here. (Python's policy towards most things). That is, trust admins to only click the button when it's valid to do so.

Reply:
That's true, though we also often make temporary branches for re-deploying docs to fix bugs, and that workflow will be simpler now.